manipulation/ManipulationPlanRequest manipulation_plan_request

---

manipulation/ManipulationPlanResponse manipulation_plan_response
